javalin/src/main/java/io/javalin/websocket/WsContext.kt

```diff
@@ -26,14 +27,29 @@ abstract class WsContext(val sessionId: String, @JvmField val session: Session)
     internal val upgradeReq by lazy { session.upgradeRequest as ServletUpgradeRequest }
     internal val upgradeCtx by lazy { upgradeReq.httpServletRequest.getAttribute(upgradeContextKey) as Context }
     internal val sessionAttributes by lazy { upgradeReq.httpServletRequest.getAttribute(upgradeSessionAttrsKey) as Map<String, Any>? }
+    internal var pingFuture : ScheduledFuture<*>? = null;
+
+    companion object {
+        var executor: ScheduledExecutorService = Executors.newSingleThreadScheduledExecutor()
+    }
```

Comment:
I wonder if there's a more natural place to put this, this doesn't feel quite right. Maybe just a custom file? We could pull out the method bodies and put them in that file?

Reply:
Sure, I'll move it